common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -691,6 +691,13 @@
   public static final String LINK = "link";
   public static final String SET_PW_LINK = "setPasswordLink";
   public static final String SUNBIRD_URL_SHORTNER_ENABLE = "sunbird_url_shortner_enable";
+  public static final String USER_PROFILE_CONFIG = "userProfileConfig";
+  public static final String USER_DATA_FIELDS = "userDataFields";
```

Comment:
userDataFields -> fields

Reply:
@bvinayakumar This key is used to fetch **userDataFields** from **userProfileConfig** in system settings. Should that be renamed to **fields**?

Reply:
yep